docs/state.md

```diff
@@ -21,6 +21,14 @@ AppStore
     startTime: number, // datetime.getTime()
     endTime: number // datetime.getTime()
   }],
+  passwords: [{
```

Comment:
on second thought, maybe this data structure should be:
```
passwords: {
  encryptedData: string, // encrypted serialization of [{origin: ..., action: ..., username:..., password: ...}]
  authTag: string,
  iv: string
}
```
so that the complete list of login details is encrypted, not just the individual passwords. This would get decrypted the first time a login detail is needed and re-encrypted every time the state is saved persistently.